src/validator/webauthn/WebAuthnFclValidator.sol

```diff
@@ -30,11 +38,18 @@ contract WebAuthnFclValidator is IKernelValidator {
 
     /// @dev The address of the p256 verifier contract (should be 0x100 on the RIP-7212 compliant chains)
     /// @dev To follow up for the deployment: https://forum.polygon.technology/t/pip-27-precompiled-for-secp256r1-curve-support/13049
-    address public immutable P256_VERIFIER;
+    address private immutable P256_VERIFIER;
+
+    /// @dev The address of the pre-compiled p256 verifier
+    address constant P256_VERIFIER_PRECOMPILED = address(0x100);
+
+    /// @dev The current p256 verifier
+    address private currentP256Verifier;
```

Comment:
iirc, this will violate the 4337 storage access rule

Reply:
Hmm I don't see why, since we are not in the context of a kernel account here, it will be called by it but that's all

Reply:
because on line 117, which is called in validateSignature, will access a storage that is not associated to account